pkg/kudoctl/packages/reader/reader_test.go

```diff
@@ -123,35 +119,29 @@ func Test_readParametersFile(t *testing.T) {
 apiVersion: kudo.dev/v1beta1
 parameters:
 `
-	param := `
+	oneParam := `
 apiVersion: kudo.dev/v1beta1
 parameters:
-  - name: example
+    - name: example
 `
-	example := make([]v1beta1.Parameter, 1)
-	example[0] = v1beta1.Parameter{Name: "example"}
+	example := []v1beta1.Parameter{{Name: "example"}}
 
-	bad := `
-apiVersion: kudo.dev/v1beta1
-parameters:
-	- oops:
```

Comment:
Fun fact: `bad` test case **was** throwing an error, but not for the reason assumed in this test! `- oops` is actually simply ignored by the unmarshaller since it is an unknown field, however it has `\t` in front of it instead of spaces and that's what was causing an error! I removed this test case since all params fields are currently `omitempty` and hence there is no real bad case to test.